pkg/cmd/server/origin/ensure.go

```diff
@@ -153,6 +153,7 @@ func (c *MasterConfig) ensureDefaultSecurityContextConstraints() {
 	sccList, err := c.KubeClient().SecurityContextConstraints().List(labels.Everything(), fields.Everything())
 	if err != nil {
 		glog.Errorf("Unable to initialize security context constraints: %v", err)
```

Comment:
Not part of the diff, but I can see it.  Tell the user why he cares that these weren't initialized.  What breaks?